java/connector-node/risingwave-connector-service/src/main/java/com/risingwave/connector/source/core/DbzCdcEventConsumer.java

```diff
@@ -55,7 +55,7 @@ public class DbzCdcEventConsumer
         // only serialize the value part
         configs.put(ConverterConfig.TYPE_CONFIG, ConverterType.VALUE.getName());
         // include record schema
-        configs.put(JsonConverterConfig.SCHEMAS_ENABLE_CONFIG, false);
+        configs.put(JsonConverterConfig.SCHEMAS_ENABLE_CONFIG, true);
```

Comment:
This line won't break the functionality, but it may hurt the performance. Because right now our source parser ignores the `schema` field emit by the Debezium framework, which means the `schema` field is useless in the JSON payload. We may need to maintain a patched version of the `JsonConverter` that eliminates the `schema` field and preserves the `payload` field in the meantime. e.g. `"payload" : { ... }`